test/parse_test.dart

```diff
@@ -83,6 +83,79 @@ void main() {
         var results = parser.parse(['--no-b']);
         expect(results['a'], isFalse);
       });
+
+      test('throws if requested as a multi-option', () {
+        var parser = ArgParser();
+        parser.addFlag('a', defaultsTo: true);
+        var results = parser.parse(['--a']);
+        throwsIllegalArg(() => results.multiOption('a'));
+      });
+    });
+
+    group('flag', () {
```

Comment:
Nit: I would do `group('flag()', () { ... })` to match other groups and make it clear this is about that method.